ios/testmanagerd/testlistener.go

```diff
@@ -0,0 +1,210 @@
+package testmanagerd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/danielpaulus/go-ios/ios/nskeyedarchiver"
+	log "github.com/sirupsen/logrus"
+)
+
+// Realtime test callbacks and test results
+type TestListener struct {
+	testFinishedChannel chan struct{}
+	err                 error
+	logWriter           io.Writer
+	debugLogWriter      io.Writer
+	testSuite           *TestSuite
```

Comment:
`testSuite` should be exposed as well, otherwise you can wait for the test to finish, but you can't access the results 😉

Reply:
Exposed.